File: 2148. Count Elements With Strictly Smaller and Greater Elements.java
class Solution {
    public int countElements(int[] nums) {
        Arrays.sort(nums);
        int maxi = nums[0] , mini = nums[0] ;
        int count = nums.length;
        for(int i:nums){
             maxi = Math.max(maxi,i);
             mini = Math.min(mini,i);
        }
        
        for(int n:nums){
            if(n==mini || n==maxi){
                count--;
            }
            
        }
        return count;
    }
}

File: 2148. Count Elements With Strictly Smaller and Greater Elements.py
class Solution:
    def countElements(self, nums: List[int]) -> int:
        p = len(nums)
        maxno = max(nums)
        minno = min(nums)
        count = 0
        for x in nums :
            if x>minno and x<maxno:
                count+=1
        return count
